diagrams/sequence-diagrams/creating-and-publishing-a-form.md

```diff
@@ -0,0 +1,28 @@
+# Creating and publishing a form
+
+```mermaid 
+sequenceDiagram
+  actor user as User
+  participant manager as Manager
+  participant designer as Designer
+  participant api as API
+
+  user -->> manager: Creates a new form
+  manager -->> api: Submit form creation information
+  api -->> manager: Returns new form information
+  manager -->> user: Redirects user to designer for new form
+  loop form creation
+    user -->> designer: Updates form
+    designer -->> api: save updated form
+    api -->> api: verify and process form
+    api -->> designer: return updated form
```

Comment:
What "updated form" is being returned? Won't it just be a `success` or `error` response?